src/react/projects/spark-core-react/src/SprkTabs/components/SprkTabsButton/SprkTabsButton.test.js

```diff
@@ -0,0 +1,36 @@
+/* global it expect describe */
+import React from 'react';
+import Enzyme, { mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import SprkTabs from '../../SprkTabs';
+import SprkTabsPanel from '../SprkTabsPanel/SprkTabsPanel';
+import SprkTabsButton from './SprkTabsButton';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe('SprkTabs Component', () => {
+  it('should display a Tab Button element with the correct base class', () => {
+    const wrapper = mount(
```

Comment:
should test SprkTabsButton on its own, without tabs / panel

Reply:
That's a good catch! I added some tests without Tabs.